### DISCUSSION TITLE
Weighted ODF issue (version 5.8 vs 5.10)

### DISCUSSION THREAD

**Discussion by lucapbelotti** [mtex-toolbox/mtex#1931] [Ask Anything] [answered]
Hi,

I want to calculate ODFs based on the mean orientation and area of the grains. To calculate this, I use the following code:
`
load("grains.mat") 
% Main data 
oris = grains.meanOrientation;
grainArea = grains.area;
CS = oris.CS;

% Calculate ODF
odf = calcDensity(oris,'weights',grainArea);
% Visualization
figure
plotPDF(odf,Miller(1,0,0,CS),'earea','minmax'); mtexColorbar`

When I run the code with versions 5.8 and 5.10, I get different results (see figure. Am I missing something?
![Comparison](https://github.com/mtex-toolbox/mtex/assets/41550623/7d19837a-f888-43e4-8d13-2f3320dd9062)

**Comment by ralfHielscher** [comment, marked as answer]
Hi,

thank you very much for reporting this issue. This is indeed a bug in MTEX 5.10. It is already fixed in develop, which we will release as MTEX 5.10.1 by the end of this week.

Ralf.

**Comment by lucapbelotti** [comment]
Thanks for the clarification Ralf.

Best,

Luca
